Lamina/include/Math/timer.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef LM_TIMER
+#define LM_TIMER
+
+#include <chrono>
+#include <iostream>
+
+#define LM_NANOSECONDS		0b00001
+#define LM_MICROSECONDS		0b00010
+#define LM_MILISECONDS		0b00100
+#define LM_SECONDS			0b01000
+#define LM_MINUTES			0b10000
+
+namespace lm
+{
+	class Timer
+	{
+	public:
+		Timer() : startTime(std::chrono::steady_clock::now()) {};
+
+		void Restart()
+		{
+			startTime = std::chrono::steady_clock::now();
+		}
+
+		float GetTime(int type)
+		{
+			float time = (float)std::chrono::duration_cast<std::chrono::nanoseconds>(std::chrono::steady_clock::now() - startTime).count();
+			switch (type)
+			{
+			case LM_NANOSECONDS:
```

Comment:
The break statements in this switch statement are redundant since the function has already returned.